src/test/cypress/e2e/exercises/quiz-exercise/QuizExerciseAssessment.cy.ts

```diff
@@ -19,43 +19,49 @@ describe('Quiz Exercise Assessment', () => {
         });
     });
 
-    describe('MC Quiz assessment', () => {
-        before('Creates a quiz and a submission', () => {
-            cy.login(admin);
-            exerciseAPIRequest.createQuizExercise({ course }, [multipleChoiceQuizTemplate], undefined, undefined, 10).then((quizResponse) => {
-                quizExercise = convertModelAfterMultiPart(quizResponse);
-                exerciseAPIRequest.setQuizVisible(quizExercise.id!);
-                exerciseAPIRequest.startQuizNow(quizExercise.id!);
+    describe(
+        'MC Quiz assessment',
+        {
+            retries: 2,
+        },
+        () => {
+            it('Assesses a mc quiz submission automatically', () => {
+                cy.login(admin);
+                exerciseAPIRequest.createQuizExercise({ course }, [multipleChoiceQuizTemplate], undefined, undefined, 10).then((quizResponse) => {
```

Comment:
As I wrote in the PR description, there is sadly no way to force a quiz to finish :/ Otherwise I definetly would have chosen that over retries. 
As the test is only flaky (fails ~ 1 out of 3 times on postgres in the cloud) I think it makes more sense to add retries as it only affects those runs and does not slow down all of them. Also it is not be clear if increasing the time would even notably affect the rate at which the tests pass.